docs/guides/Integration-Guides/erp-integration-guide/order-modifications.md

```diff
@@ -1,28 +1,29 @@
 ---
-title: "Change order"
-slug: "change-order"
+title: "Order modifications"
+slug: "order-modifications"
```

Comment:
Please remember to open a pull request on devportal to create a redirect from the old URL to the new URL